constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayoutBaseScope.kt

```diff
@@ -678,6 +678,463 @@ abstract class ConstraintLayoutBaseScope internal constructor(extendFrom: CLObje
         return ref
     }
 
+    /**
+     * Grid helpers make build a Row representation more intuitively.
```

Comment:
I think this should be more like:

Creates a Grid based helper that lays out its elements in a single Row.